socorro/cron/jobs/update_signatures.py

```diff
@@ -48,7 +47,8 @@ class UpdateSignaturesCronApp(BaseCronApp):
     )
     required_config.add_option(
         'elasticsearch_class',
-        default=ElasticsearchConfig,
+        default='socorro.external.es.connection_context.ConnectionContext',
```

Comment:
In configuration, "class" indicates a Python dotted module name--not an actual class. I fixed this to take a Python dotted module name.